packages/core-ui/styles/buttons.scss

```diff
@@ -322,4 +322,19 @@ SPDX-License-Identifier: Apache-2.0 */
   &.active {
     background: var(--button-secondary-background);
   }
+
+  > div {
+    @include btn-small;
+
+    margin-left: var(--button-spacing-large);
+    background: var(--network-color-secondary);
+    font-size: var(--button-font-size-small);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    min-width: 20px;
```

Comment:
Is this necessary? 

Is `max-width` needed? Or shall we just do trimming on the badge text before it is passed into the component?